Mac/AppDelegate.swift

```diff
@@ -921,16 +919,16 @@ internal extension AppDelegate {
 			  let window = mainWindowController?.window else {
 				  return
 			  }
-		
+		themeImportPath = userInfo["path"] as? String
 		var informativeText: String = ""
 		if let decodingError = error as? DecodingError {
 			switch decodingError {
 			case .typeMismatch(let type, _):
-				informativeText = "Type '\(type)' mismatch."
+				informativeText = "the type—'\(type)'—is mismatched."
 			case .valueNotFound(let value, _):
-				informativeText = "Value '\(value)' not found."
+				informativeText = "the value—'\(value)'—is missing in the Info.plist."
 			case .keyNotFound(let codingKey, _):
-				informativeText = "Key '\(codingKey.stringValue)' not found."
+				informativeText = "the key—'\(codingKey.stringValue)'—is missing in the Info.plist."
 			case .dataCorrupted( _):
 				informativeText = error.localizedDescription
 			default:
```

Comment:
We should use `NSString.localizedStringWithFormat` here instead of string interpolation in case we do end up localizing the AppKit version of the app. I would normally not worry about it, but this would be easily overlooked if we did do localization on the AppKit version.

Reply:
Sorted.